main/lsp/wrapper.h

```diff
@@ -29,7 +26,8 @@ class LSPWrapper final {
     std::unique_ptr<WorkerPool> workers;
     std::shared_ptr<spd::sinks::ansicolor_stderr_sink_mt> stderrColorSink;
     std::shared_ptr<spd::logger> typeErrorsConsole;
-    std::unique_ptr<LSPOutputToVector> output;
+    std::shared_ptr<LSPConfiguration> config;
+    std::shared_ptr<LSPOutputToVector> output;
```

Comment:
While `output` is present on `config`, `config` has the reference as `shared_ptr<LSPOutput>`. I retain a copy here so I can easily dump out messages it contains w/o casting.